resources/js/directives.js

```diff
@@ -0,0 +1,10 @@
+import validateElementVAT from './vat-validation'
+
+Vue.directive('validate', {
+    bind(el, binding) {
+        if ('vat' in binding.modifiers) {
+            el.addEventListener('change', (event) => validateElementVAT(event.target))
```

Comment:
This would leave dangling eventlisteners with turbo navigations
https://vueuse.org/core/useEventListener/